app/src/main/res/drawable/splash_background.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@drawable/bg_gradient"/>
+    <item>
+        <bitmap android:gravity="center"
+            android:src="@drawable/iou_logo" />
```

Comment:
Looking at the splash screen, maybe we can use a png with transparent background haha